src/web/app/pages-student/student-course-details-page/student-course-details-page.component.spec.ts

```diff
@@ -27,4 +28,57 @@ describe('StudentCourseDetailsPageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with populated fields', () => {
+    const courseAttributes: any = {
+      createdAt: '1552509888215',
```

Comment:
LGTM with a very minor nit: Maybe the `createdAt` value can follow this format: "2019-03-12T09:13:29.272Z" instead of using a unix timestamp value

Reply:
Gotcha :)